src/__tests__/componentsTests/Destinations.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer, { create } from 'react-test-renderer';
+
+import Destinations from '../../components/Destinations';
+
+describe('Destinations', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(<Destinations />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+  test('the className of the component includes container', () => {
+    const { root } = create(<Destinations />);
+    const element = root.findByType('div');
+    expect(element.props.className.includes('container')).toBe(true);
+  });
+  test('the  component to have h4 element', () => {
+    const { root } = create(<Destinations />);
+    const element = root.findByType('h4');
+    expect(element.type).toBe('h4');
+  });
+  test('the  component to have h5 element with right content', () => {
+    const { root } = create(<Destinations />);
+    const element = root.findByType('h5');
+    expect(element.children.includes(' Please select Favourite Destination ')).toBe(true);
+  });
+});
```

Comment:
[OPTIONAL] When doing tests always consider positive and negative scenarios not only positive. You'd be surprised how doing this can catch unforeseen errors. Does it behave the way you expect in a negative scenario?